pkg/internal/common/logging/metadata/level.go

```diff
@@ -6,26 +6,34 @@ SPDX-License-Identifier: Apache-2.0
 
 package metadata
 
-import (
-	"github.com/hyperledger/aries-framework-go/pkg/common/log"
+// Level defines all available log levels for logging messages.
+type Level int
+
+// Log levels.
+const (
```

Comment:
I assume this was duplicated to avoid circular references?

This should have a comment that it needs to stay in sync with the other package.

Reply:
Will add those comments.